src/Arcus.WebApi.OpenApi.Extensions/Arcus.WebApi.OpenApi.Extensions.csproj

```
@@ -14,11 +14,14 @@
     <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <GenerateOpenApiDocuments>false</GenerateOpenApiDocuments>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Guard.Net" Version="1.2.0" />
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc4" Condition="'$(TargetFramework)' == 'netcoreapp3.0'" />
```

It got bumped to RC 5 yesterday, would you mind updating it please?
```suggestion
    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc5" Condition="'$(TargetFramework)' == 'netcoreapp3.0'" />
```